full-service/src/json_rpc/wallet.rs

```diff
@@ -806,12 +862,70 @@ where
                 txo_map,
             }
         }
+        JsonCommandRequest::get_txos_for_view_only_account {
+            account_id,
+            offset,
+            limit,
+        } => {
+            // TODO(cc) share logic between this and regular account get txos
+            let o = offset.parse::<i64>().map_err(format_error)?;
+            let l = limit.parse::<i64>().map_err(format_error)?;
```

Comment:
IIUC these should use `format_invalid_request_error`

Reply:
yes!